feature-libs/product-configurator/rulebased/components/config/configurator-ui-settings.ts

```diff
@@ -0,0 +1,13 @@
+export abstract class ConfiguratorUISettings {
```

Comment:
1. Not to pollute the root config with too many properties, let's have one config property for the whole configurator library with subconfigs for specific features - 
```
productConfigurator: {
   rulebased: {
      quantityDebounceTime?: number;
      inputDebounceTime?: number;
   }
}
```
2. Preceed any config the class with (to be auto-provided)
```
@Injectable({
  providedIn: 'root',
  useExisting: Config,
})
```
3. Suffix the class name with `...Config`

Reply:
pick

Reply:
@Uli-Tiger `debounceTime` as a key sounds a bit too general. I guess we're debouncing the update. In that case, what do you think about `updateDebounceTime`?

Reply:
@Uli-Tiger Did you address this comment also?

Reply:
done